src/modules/draft/draftApi.js

```diff
@@ -107,6 +108,24 @@ export const fetchCompetences = query =>
     resolveJsonOrRejectWithError,
   );
 
+export const fetchCompetenceTitle = async competenceCode => {
+  let url;
+  if (competenceCode[1] === 'E') {
+    url = grepUrl(`/kjerneelementer-lk20/${competenceCode}`);
+  } else if (competenceCode[1] === 'M') {
+    url = grepUrl(`/kompetansemaal-lk20/${competenceCode}`);
+  } else {
+    url = grepUrl(`/kompetansemaalsett-lk20/${competenceCode}`);
+  }
+
+  const response = await fetchAuthorized(url);
+  const jsonResponse = await response.json();
```

Comment:
Stemmer for å bruke funksjonaliteten vi allerede har for å resolve json
`fetchAuthorized(...).then(resolveJsonOrRejectWithError)`